apps/typeform/lambda/serverless.yml

```diff
@@ -0,0 +1,46 @@
+service:
+  name: sls-apps-typeform
+
+frameworkVersion: '>=1.0.0 <2.0.0'
+
+plugins:
+  - serverless-domain-manager
+  - serverless-express
```

Comment:
I don't think it's needed. As a matter of fact you don't have the plugin installed. https://github.com/awslabs/aws-serverless-express we're using doesn't require any plugins